portfolio/src/main/webapp/index.html

```diff
@@ -41,6 +41,7 @@
           <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#comments">Comments</a></li>
         </ul>
       </div>
+      <a class="nav-link js-scroll-trigger" href="#" id="login">LOGIN</a>
```

Comment:
since you already have `text-transform: uppercase`, typing all uppercase here is redundant. I could see an argument to leaving this as `Login`, and you can change it in the stylesheet as appropriate, or you can remove the CSS version. Up to you!